Description: Cleanup and ready for PPA
 Rebuilt the package from github files. Uploading as a PPA.
 .
 dhis2-tools (1.2-1ubuntu1) trusty; urgency=low
 .
   * Rebuilt from github code. Ready for PPA upload.
Author: Simon Jespersen <simjes91@me.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ dhis2-tools-1.2/Makefile
@@ -0,0 +1,11 @@
+build:
+	for subd in backups lib  logs  temp  webapps; do \
+		mkdir -p ./usr/share/dhis2-tools/skel/$$subd; \
+	done	
+	
+	mkdir -p ./var/lib/dhis2;
+	mkdir -p ./var/log/dhis2;
+
+	#perimissions
+	find ./usr -type d -exec chmod 0755 {} \;
+	find ./usr -type f -exec chmod 0644 {} \;
